### DISCUSSION TITLE
Multiple users and email matching

### DISCUSSION THREAD

**Discussion by TheLastProject** [jonaswinkler/paperless-ng#275] [Support] [answered]
Hey,

Wow, this is one awesome tool.

I was thinking of giving my parents access to my instance as well, but of course I don't want them to see all my documents (and I don't care to see theirs). Right now I use an email address as incoming and I tag it with my name but I was wondering... is there any way to enforce it to be saved to a certain user's documents or am I understanding the purpose of users wrong perhaps?

**Comment by shamoon** [comment, marked as answer]
Agreed, a more granular user system is certainly a useful thing but given the project is still pre-1.0 and this is a not-insignificant undertaking, Im not sure its on the roadmap just yet. Of course I wont speak for the project owner but thats my assumption having been involved for a bit.

**Reply by TheLastProject** [reply]
I wonder, what are users and groups used for currently? Is there already some separation in any form or can everyone access everything?

**Reply by jonaswinkler** [reply]
Everyone can access everything.

Users and groups are there simply because the framework used for this project has built-in support for that, and I didn't see a reason to disable it.

See also #52.

I'm considering that, but as of now there are more pressing issues.

**Reply by shamoon** [reply]
Ah yes, that’s the # I was looking for.
